service/templates/admin.html

```diff
@@ -365,6 +365,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
                     <label>Task finder</label>
                     <input type="text" ng-model="Settings.scheduler.task_finder">
                   </md-input-container>
+                  <md-input-container class="control" style="width:45%; margin-left:50px;">
```

Comment:
leave off the margin-left:50px
Those are used for controls on the right side to space things out